src/configurations/destinations/zoho_dev/accounts/zoho_dev_oauth/db-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "DESTINATION_ZOHO_DEV_OAUTH",
+  "type": "zoho_dev",
+  "category": "destination",
+  "authenticationType": "oauth",
+  "config": {
+    "optionFields": ["region"],
+    "oauth": {
+      "generateToken": true,
+      "addTokenRefresh": true
```

Comment:
@utsabc are we going ahead with same variable names ?

Reply:
If you still want to highlight refresh & adding token capability, you can find alternative to `addTokenRefresh` something on the lines of `manageTokenInDataplane` or to keep it simple `refreshTokenInDataplane`

Reply:
Yeah lets put in the thread I will update the design doc and @ItsSudip can update this PR.
I am ok with `refreshTokenInDataplane`